src/OpenTelemetry.Exporter.ZPages/ZPagesExporterStatsHttpServer.cs

```diff
@@ -0,0 +1,143 @@
+﻿// <copyright file="ZPagesExporterStatsHttpServer.cs" company="OpenTelemetry Authors">
+// Copyright 2018, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using System.IO;
+using System.Net;
+using System.Threading;
+using System.Threading.Tasks;
+using OpenTelemetry.Exporter.ZPages.Implementation;
+using OpenTelemetry.Trace.Export;
+
+namespace OpenTelemetry.Exporter.ZPages
+{
+    /// <summary>
+    /// A HTTP listener used to expose ZPages stats.
+    /// </summary>
+    public class ZPagesExporterStatsHttpServer : IDisposable
```

Comment:
This looks very similar to the Prometheus http server - maybe we could refactor it to be exporter agnostic and put it into a shared library?

Reply:
good suggestion. Perhaps a spearate work item. @n1dtec can you please create an issue to track it?

Reply:
Created an issue for tracking this : https://github.com/open-telemetry/opentelemetry-dotnet/issues/476